R/readTimeseries.R

```diff
@@ -25,7 +28,25 @@ readTimeSeries <- function(series, con,
                            schema = "timeseries",
                            env = NULL,
                            pkg_for_irreg = "xts",
-                           chunksize = 10000){
+                           chunksize = 10000,
+                           regex = FALSE){
+  
+  if(regex) {
+    if(length(series) > 1) {
+      stop("Only supports a single expression in series!")
+    }
+    
+    pattern <- series
+    
+    match_query <- sprintf("SELECT ts_key FROM %s.timeseries_main WHERE ts_key ~ '%s'",
```

Comment:
Did you try do run this in SQL? You should definitely get more than one key back.